gulpfile.babel.js

```diff
@@ -267,7 +267,6 @@ gulp.task('metadata', () => {
 gulp.task('default', ['clean'], cb => {
   runSequence(
     ['styles', 'styles-grid'],
-    ['scripts'],
```

Comment:
@contra actually offered his help a good while ago, but I didn’t want to waste his time on the current state of the gulpfile but rather the rewrite. I guess I’ll take that on as my next big thing :tm: